src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck(String machineType) {
+        super(machineType);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck" + " " + machineType + " " + "started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck" + " " + machineType + " " + "stopped working. ");

```

Comment:
The stop message should also be informative and properly formatted. Consider changing it to: `System.out.println("Truck " + machineType + " stopped working.");`